homeassistant/components/systemmonitor/sensor.py

```diff
@@ -316,9 +316,11 @@ def update(self):
             else:
                 self._state = None
         elif self.type == "last_boot":
-            self._state = dt_util.as_local(
-                dt_util.utc_from_timestamp(psutil.boot_time())
-            ).isoformat()
+            # Only update on initial setup
+            if self._state is None:
+                self._state = dt_util.as_local(
```

Comment:
All times in the state machine should be UTC times. We should fix this in a future PR.